protocol/communities/persistence.go

```diff
@@ -374,7 +374,7 @@ func (p *Persistence) SaveRequestToJoin(request *RequestToJoin) (err error) {
 		return ErrOldRequestToJoin
 	}
 
-	_, err = tx.Exec(`INSERT INTO communities_requests_to_join(id,public_key,clock,ens_name,chat_id,community_id,state) VALUES (?, ?, ?, ?, ?, ?, ?)`, request.ID, request.PublicKey, request.Clock, request.ENSName, request.ChatID, request.CommunityID, request.State)
+	_, err = tx.Exec(`INSERT OR REPLACE INTO communities_requests_to_join(id,public_key,clock,ens_name,chat_id,community_id,state) VALUES (?, ?, ?, ?, ?, ?, ?)`, request.ID, request.PublicKey, request.Clock, request.ENSName, request.ChatID, request.CommunityID, request.State)
```

Comment:
This is needed so that we don't have multiple times the same request.

Reply:
Isn't it already solved by: `PRIMARY KEY (id) ON CONFLICT REPLACE`?

Reply:
```
CREATE TABLE communities_requests_to_join  (
  id BLOB NOT NULL,
  public_key VARCHAR NOT NULL,
  clock INT NOT NULL,
  ens_name VARCHAR NOT NULL DEFAULT "",
  chat_id VARCHAR NOT NULL DEFAULT "",
  community_id BLOB NOT NULL,
  state INT NOT NULL DEFAULT 0,
  PRIMARY KEY (id) ON CONFLICT REPLACE <-----
);
```

Yes, I don't think we need that.

Reply:
Yeah. 
I think I still prefer the query with the `INSERT OR REPLACE` though, because it makes it explicit that we will replace the old one if it exists, without having to check the `CREATE TABLE ` statement.